packages/immutable-rootfs/30cos-immutable-rootfs/cos-mount-layout.sh

```diff
@@ -177,6 +178,7 @@ function mountState {
     local fstab_line
     local state_dir
 
+    [ ! -d "${mount}" ] && echo "Source mount path ${mount} does not exist!" && return
```

Comment:
Same here as before, we should test against `${base}` and only check inside the `"${state_bind}" = "true"` path. Again we should not `echo` any message in this method. Line 187 `mkdir -p "${base}" "${state_dir}"` can be changed to `mkdir -p "${state_dir}"`, as we will know `${base}` exists at this stage.